### PR TITLE
Add a Pedigree panel in STR page

### PR DESCRIPTION
**Pull request by moedarrah**
This PR adds:
A Pedigree panel on the STR page when the length of (case.madeline_info & case.individuals) > 1

Screen Shot 2021-05-20 at 14 03 01

**Review:**
- [x] code approved by DN
- [x] tests executed by MOD

### CODE REVIEW

File: scout/server/blueprints/variants/templates/variants/str-variants.html [hunk 1]
```diff
@@ -121,6 +121,13 @@
 <div class="container-fluid">
   {{ pagination_footer(more_variants, page) }}
 </div>
+<!--  Pedigree_panel -->
+<div class="row">
```
Comment (dnil) [inline scout/server/blueprints/variants/templates/variants/str-variants.html:125]: You tell me what is best practice, but I probably would have kept the row div within the if statement for now?

File: scout/server/blueprints/variants/templates/variants/str-variants.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {% extends "layout_bs4.html" %}
-{% from "utils.html" import comments_table %}
+{% from "utils.html" import comments_table, pedigree_panel %}
```
Comment (dnil) [inline scout/server/blueprints/variants/templates/variants/str-variants.html:2]: 👍

**Review by dnil — APPROVED** [review]

File: scout/server/blueprints/variants/templates/variants/str-variants.html (hunk 1 above)
Comment (moedarrah) [reply scout/server/blueprints/variants/templates/variants/str-variants.html:125]: Totally agree. Thanks

File: scout/server/blueprints/variants/templates/variants/str-variants.html [hunk 3]
```diff
@@ -121,6 +121,13 @@
 <div class="container-fluid">
   {{ pagination_footer(more_variants, page) }}
 </div>
+<!--  Pedigree_panel -->
+{% set has_pedigree = case.madeline_info and case.individuals|length > 1 %}
+{% if has_pedigree %}
+<div class="row">
+    <div class="col-md-4">{{ pedigree_panel(case) }}</div>
+  {% endif %}
```
Comment (dnil) [inline scout/server/blueprints/variants/templates/variants/str-variants.html:129]: hihi almost - the end div needs to go inside as well.. 😉

Comment (moedarrah) [reply scout/server/blueprints/variants/templates/variants/str-variants.html]: 😳

**Comment by dnil** [comment]
Thank you!

**Pull request merged**